docs/calicoctl/bgp.md

```diff
@@ -0,0 +1,218 @@
+
+# User guide for 'calicoctl bgp' commands
+
+This sections describes the `calicoctl bgp` commands.
+
+These commands can be used to manage the global BGP configuration, including:
+ - global BGP peers (the BGP speakers that peer with every Calico node in the 
+   network)
+ - default values to use for the AS number
+ - whether a full BGP mesh is required between all of the Calico nodes
+
+Calico node specific BGP configuration, such as having BGP peers specific to a 
+particular Calico node, is configured using the `calicoctl node` commands. You 
+can read about these commands in the [`calicoctl node` guide](./node.md).
+
+For an overviw of BGP configuration, read the [BGP guide](../bgp.md), which
+covers in more detail all available BGP related commands, including use cases.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a 
+full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl bgp' commands
+
+Run `calicoctl bgp --help` to display the following help menu for the 
+calicoctl bgp commands.
+
+```
+
+Usage:
+  calicoctl bgp peer add <PEER_IP> as <AS_NUM>
+  calicoctl bgp peer remove <PEER_IP>
+  calicoctl bgp peer show [--ipv4 | --ipv6]
+  calicoctl bgp node-mesh [on|off]
+  calicoctl bgp default-node-as [<AS_NUM>]
+
+
+Description:
+  Configure default global BGP settings for all nodes. Note: per-node settings
+  will override these globals for that node.
+
+Options:
+ --ipv4    Show IPv4 information only.
+ --ipv6    Show IPv6 information only.
+
+```
+
+## calicoctl bgp commands
+
+
+### calicoctl bgp peer add <PEER_IP> as <AS_NUM>
+This command is used to add a global BGP peer.
+
+The command can be run on any Calico node.
```

Comment:
It's more permissive even than this: it can be run anywhere that can modify the etcd datastore.